src/sender.js

```diff
@@ -36,7 +38,16 @@ async function initialize() {
     chainId = await getChainId(web3Instance)
     connectSenderToQueue({
       queueName: config.queue,
-      cb: main
+      cb: options => {
+        if (maxProcessingTime) {
+          return watchdog(() => main(options), maxProcessingTime, () => {
+            logger.fatal('Max processing time reached')
+            process.exit(EXIT_CODES.MAX_TIME_REACHED)
+          })
+        }
+
+        return main(options)
+      }
     })
   } catch (e) {
     logger.error(e.message)
```

Comment:
Should `process.exit(1)` also use a constant from `EXIT_CODES`?

Reply:
Done in 3d706ddaf7e108b1ce2fe4756364236d845b7a24.